common/sai_client/sai_redis_client/sai_redis_client.py

```diff
@@ -745,8 +749,17 @@ def __operate(self, obj, attrs, op):
         # Make redis-style OIDs
         obj = obj.replace('oid:0x', '0x')
         obj = obj.replace('0x', 'oid:0x')
-        attrs = attrs.replace('oid:0x', '0x')
+
+        # The following logic adds a prefix "oid:" to odject ID it attrs
+        # if there are no this prefix. And there is a workaround
+        # for cases when the inserts "oid:" incorrectly.
+        # There must be an issue on github to replace it with
+        # a normal logic
+        attrs = attrs.replace('oid:0x', 'OID_HERE')
```

Comment:
Why do we need this? I believe this will not work in some scenarios... E.g., sairedis.rec or SAI CLI